SalesforceCommerceCloud/pwa-kit
# Update translations

* Updated translations
  * Changed "here to help" message IDs
* Removed some hard-coded text
  * Slightly rekerjiggered how American Express tooltip is generated
* Changed checkout error from API response (not localized) to generic message (localized)
  * Will have follow-up PR to add localized API responses

# Description



# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- (change1)

# How to Test-Drive This PR

Start the server and use your preferred non-English locale.
- Find a UI with updated text. Load it. Observe the text.

Start the server with the pseudolocale.
- Bottom of the home page, observe that the text above the "Contact Us" button is localized.
- Add an item to your wishlist. In the green confirmation popup, observe that the "View" link is localized.
- View your wishlist. Observe that the "Select Options" button is localized.
- Buy an item. Before entering credit card details, hover over the security code tooltip. Observe that it is localized.
- Enter an American Express credit card. Hover over the security code tooltip. Observe that it is localized and not the same as the default text.
- Attempt to check out with no items in cart. Observe localized generic error message.

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
